### DISCUSSION TITLE
authenticator.cc browser extensions

### DISCUSSION THREAD

**Discussion by ymk92** [Authenticator-Extension/Authenticator#936] [Q&A] [answered]
Dear all, 

We have some questions on the authenticator.cc browser extensions: 

-	The setup of the extension itself ? are there any cloud services involved?
-	MS Edge is our default browser and will be updated regular – would that extension create any problems for upgrading the browser? (same for Google Chrome)
-	Is the extension getting constant updates ? is there support for the extension (especially as it is providing key security features)?
-	How is this extension managed? How is it upgraded; who is owning the lifecycle of this software extension?
-	How is it ensured that the extension data is not synced?

**Comment by Sneezry** [comment, marked as answer]
Hi @ymk92 

Thanks for reaching out.

> The setup of the extension itself ? are there any cloud services involved?

We (this extension developers) don't own any cloud services for this extension, but some cloud services may be involved based on your settings:

1. Browser sync. If you enable browser sync (this is enabled by default, you can disable it in settings, preferences page), your data will be uploaded to browser built-in service. Different service will be used based on the browse you use, see details for [Chrome](https://support.google.com/chrome/answer/185277?hl=en&co=GENIE.Platform%3DDesktop), [Firefox](https://www.mozilla.org/en-US/firefox/sync/) and [Edge](https://support.microsoft.com/en-us/microsoft-edge/sign-in-to-sync-microsoft-edge-across-devices-e6ffa79b-ed52-aa32-47e2-5d5597fe4674#:~:text=Sync%20your%20account%20on%20your%20computer&text=Select%20the%20profile%20image%20in,you're%20already%20signed%20in.)
2. Auto backup to cloud storage. This is disabled by default, and you need to sign in 3rd party cloud storage service to enable it. We support Google Drive, Dropbox and Microsoft OneDrive currently.

> MS Edge is our default browser and will be updated regular – would that extension create any problems for upgrading the browser? (same for Google Chrome)

No, browser extensions won't block browser upgrading.

> Is the extension getting constant updates ? is there support for the extension (especially as it is providing key security features)?

Yes, we are working on this extension, and will release new versions for cool features and urgent fixes.

To be honest, the support is limited, the most important reason is maintainers have their own jobs. Also, we cannot give any help for some issues like data missing because we don't collect any data. Backup is always the only thing can save you.

However, we are still glade to help for questions of the extension.

> How is this extension managed? How is it upgraded; who is owning the lifecycle of this software extension?

The extension is published to browser extension marketplace, it will auto upgrade if a new version releases. This is managed by the browser itself. We (the extension developers) own the lifecycle of this extension, and every version will be reviewed by the marketplace before it is available to the public.

> How is it ensured that the extension data is not synced?

Disable browser sync and cloud storage backup will prevent the extension syncing any data.

If you install the extension with enterprise policy, you can also disable these settings with the policy:

* Disable browser sync: https://github.com/Authenticator-Extension/Authenticator/blob/release/manifests/schema-chrome.json#L19
* Disable cloud storage backup: https://github.com/Authenticator-Extension/Authenticator/blob/release/manifests/schema-chrome.json#L9

For enterprise policy configuration, see https://authenticator.cc/docs/en/policy-config

For Edge, see https://docs.microsoft.com/en-us/deployedge/microsoft-edge-manage-extensions

**Reply by ymk92** [reply]
Thanks for your prompt reply and those answers are really helpful!
